### DISCUSSION TITLE
Problem with many simultaneous connections

### DISCUSSION THREAD

**Discussion by jflmateusnassar** [moquette-io/moquette#759] [Q&A]
I'm trying to use moquette for heavy connections. 200k connections or more. When starting the moquette, almost 60 thousand connections go up in a time interval of 2.5 minutes. In this scenario, the moquette does not present any failures, but I verified that the methods onConnect, onPublish are not called when a high number of connections go up in the moquette and everything stops working, but without showing errors.
I'm using linux, with aws ec2 machine with 8 CPUs and 32G of RAM.
What could be happening? Would you help me?

**Comment by andsel** [comment]
Hi @jflmateusnassar to check what's happening we need some information.
- the settings used to launch the JVM, for example `Xmx`etc
- a heap dump (use the `jhat`tool to grab it) when the problem is not present and when it's present.
- the status of the connections when this happen, you can use `netstat -anv -p tcp | grep <pid>` or `nettop -p <pid>` (to discover the pid, use `jps`). If TCP permit to open at maximum 65K port, how did you reached the 200k connections test?
